docusaurus.config.js

```diff
@@ -67,14 +87,28 @@ const config = process.env.IASQL_ENV === 'local' ? Object.assign(baseConfig, loc
                 href: 'https://iasql.com',
                 target: '_self',
               },
+              {
+                label: 'Docs',
+                href: 'https://docs.iasql.com',
+                target: '_self',
+              },
+              {
+                label: 'GitHub',
+                href: 'https://github.com/iasql/iasql-engine',
+                target: '_self',
+              },
             ],
           },
           {
             title: 'Community',
             items: [
               {
-                label: 'Reddit',
-                href: 'https://reddit.com/r/iasql',
```

Comment:
We're dropping Reddit?

Reply:
Talked about offline. We are going to focus on Discord and GH discussions/issues for now